orbs.md

```diff
@@ -0,0 +1,211 @@
+# Orbs + Sample of Orb
+
+## Orbs
+**CircleCI Orbs** are shareable packages of configuration elements, including jobs, commands, and executors. CircleCI provides certified orbs, along with 3rd-party orbs authored by CircleCI partners. It is best practice to first evaluate whether any of these existing orbs will help you in your configuration workflow. Refer to the [CircleCI Orbs Registry](https://circleci.com/orbs/registry/) for the complete list of certified orbs.
+
+## Design Methodology
+Before using orbs, you may find it helpful to understand the various design decisions and methodologies that were used when these orbs were designed. Orbs were designed with the following considerations:
+
+- Orbs are transparent - If you can execute an orb, you and anyone else can view the source of that orb.
+
+- Metadata is available - Every key can include a description key and an orb may include a description at the top level.
+
+- Production orbs are always semantic versioned (semver’d) - CircleCI allows development orbs that have versions that start with dev:.
+
+- Production orbs are immutable - Once an orb has been published to a semantic version, the orb cannot be changed. This prevents unexpected breakage or changing behaviors in core orchestration
+
+- One registry (per install) - Each installation of CircleCI, including circleci.com, has only one registry where orbs can be kept.
+
+## Orb Structure
+
+Orbs consist of the following elements:
+
+- Commands
+- Jobs
+- Executors
```

Comment:
I am not sure if we need these elements description in this part of our tutorial. As I understand, this staff not specific for orbs, it's similar everywhere. `commands` description already exists in `version_2_1.md` for example.